<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-MRVJBNT" failures="0" tests="1" name="com.menumitratCommonAPITestScript.UpdateOrderPaymentSettleTestScript" time="0.279" errors="1" timestamp="23 Jun 2025 07:21:44 GMT">
  <testcase classname="com.menumitratCommonAPITestScript.UpdateOrderPaymentSettleTestScript" name="orderPaymentSettleTest" time="0.279">
    <error message="
The data provider is trying to pass 8 parameters but the method com.menumitratCommonAPITestScript.UpdateOrderPaymentSettleTestScript#orderPaymentSettleTest takes 9 and TestNG is unable in inject a suitable object" type="org.testng.TestNGException">
      <![CDATA[org.testng.TestNGException: 
The data provider is trying to pass 8 parameters but the method com.menumitratCommonAPITestScript.UpdateOrderPaymentSettleTestScript#orderPaymentSettleTest takes 9 and TestNG is unable in inject a suitable object
	at org.testng.internal.Invoker.injectParameters(Invoker.java:1253)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1122)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
Caused by: java.lang.ArrayIndexOutOfBoundsException: Index 8 out of bounds for length 8
	at org.testng.internal.Invoker.injectParameters(Invoker.java:1249)
	... 17 more
]]>
    </error>
  </testcase> <!-- orderPaymentSettleTest -->
</testsuite> <!-- com.menumitratCommonAPITestScript.UpdateOrderPaymentSettleTestScript -->
